### ISSUE TITLE
Broken OpenBSD 7 image

### ISSUE THREAD

**Issue by TheTumultuousUnicornOfDarkness** [hummeltech/openbsd-vagrant-action#1]
Hello,

The `generic/openbsd7` image created by Roboxes is broken since some weeks: this a an OpenBSD 7.4 system, but the configured mirror in the image (https://ftp.usa.openbsd.org/pub/OpenBSD/) does not contain 7.4 packages anymore.

This is the error I have when I use this GitHub Action:
```
2024-11-26T10:18:33.2985951Z ==> default: Machine booted and ready!
2024-11-26T10:18:33.7477647Z ==> default: Running provisioner: shell...
2024-11-26T10:18:36.8558700Z     default: Running: inline script
2024-11-26T10:18:38.7223074Z     default: https://ftp.usa.openbsd.org/pub/OpenBSD//7.4/packages/amd64/: no such dir
2024-11-26T10:18:38.7252294Z     default: Can't find bash
2024-11-26T10:18:38.7401425Z     default: Can't find rsync--
```
There is an issue opened upstream about this: lavabit/robox#305.

As suggested by borgbackup/borg#8554, you can fix that by adding `echo "https://ftp.eu.openbsd.org/pub/OpenBSD" > /etc/installurl` before this line:
https://github.com/hummeltech/openbsd-vagrant-action/blob/587c79f15de4747b96fdfc525ff32955988e3376/action.yml#L50
This mirror has packages for all OpenBSD versions, so it should fix the issue and avoid the same issue in the future (e.g. when OpenBSD 7.7 will be released and OpenBSD 7.5 will become EOL).

**Comment by hummeltech** [comment]
Thanks for reporting this and for the research. I have pushed up a "fix" for this issue and released it as `v3.0`, along with another update to the parent `vagrant-action` to support `VirtualBox` when using `Ubuntu`, which really beefs up the number of different box images which can be used.

**Comment by TheTumultuousUnicornOfDarkness** [comment]
```
 ==> default: Running provisioner: shell...
    default: Running: inline script
    default: Update candidates: quirks-6.160 -> quirks-6.160
    default: quirks-6.160->6.160 (processing)
    default: quirks-6.160 signed on 2024-04-02T23:15:13Z
    default: bash-5.2.15 (processing)
    default: rsync-3.2.7p1 (processing)
    default: rsync-3.2.7p1 (extracting)
    default: rsync-3.2.7p1 (skipping)
    default: rsync-3.2.7p1 (installing)
    default: Adding rsync-3.2.7p1
    default: The following new rcscripts were installed: /etc/rc.d/rsyncd
    default: See rcctl(8) for details.
    default: sshd(ok)
    default: sshd(ok)
```
That is better now with the latest version, thanks!
